connection.php

```diff
@@ -0,0 +1,725 @@
+<?php
+
+/**
+* @author: mfsi-krishnadev
+* @class : database connection and operation
+* 
+*/
+class db_connection 
+{
+	//declaration of variables
+	private $hostname = "localhost";
+	private $username = "root";
+	private $password = "mindfire";
+	private $database = "employee";
+	public  $connect;
+
+	//contructor to initialse the variable
+	function __construct()
+	{
+		$this->connect = mysqli_connect($this->hostname,$this->username,$this->password,$this->database);
+
+		//Error Handling
+		if(mysqli_connect_error())
+		{
+			trigger_error("Failed to connect mySQL:". mysqli_connect_error(),E_USER_ERROR);
+		}	 
+	}
+
+	/*
+	 * @access: public
+	 * @param : integer
+	 * @return type: array
+	*/
+	function display()
+	{
+			$conn = $this->connect;
+
+				$display = "SELECT CONCAT(prefix, ' ',first_name, ' ',middle_name , ' ', last_name)as name, gender, email_id, dob, marital_status, id,
+					(SELECT GROUP_CONCAT(street, ',',city, ',',state, '-',zip) AS residence FROM address addr WHERE type = 'residence' AND addr.employee_id = e.id)as residence,
+					(SELECT GROUP_CONCAT(street, ',',city, ',',state, '-',zip) AS office FROM address addr WHERE type = 'office' AND addr.employee_id = e.id)as office,
+					(SELECT type FROM communication commu  WHERE commu.employee_id = e.id )as communication
+					FROM employee e " ;
+
+		 	$result = mysqli_query($conn,$display);
+		 	return $result;
+	}
+
+	/*
+	 * @access: public
+	 * @param: array
+	 * @return : integer
+	*/
+
+	function insert($input = array())
```

Comment:
It was discussed in person to change the function have it more generic so that this function can be used for any table insert operation.
Right now it is not and specific insert queries for employee, address and other tables are written here.